sycl/test/basic_tests/image_accessor_types.cpp

```diff
@@ -0,0 +1,29 @@
+// RUN: not %clangxx -fsyntax-only %s 2>&1 | FileCheck %s
+#include <CL/sycl.hpp>
+#include <iostream>
+
+using namespace cl::sycl;
+class mod_image;
+int main() {
+  half4 src[16];
+  image<2> srcImage(src, image_channel_order::rgba, image_channel_type::fp16,
+                    range<2>(4, 4));
+  queue myQueue;
+  myQueue.submit([&](handler &cgh) {
+    accessor<float, 2, access::mode::read, access::target::image> NotValidType1(
+        srcImage, cgh);
+    // CHECK: The data type of an image accessor must be only cl_int4, cl_uint4, cl_float4 or cl_half4
+    accessor<int2, 2, access::mode::read, access::target::image> NotValidType2(
+        srcImage, cgh);
+    // CHECK: The data type of an image accessor must be only cl_int4, cl_uint4, cl_float4 or cl_half4
+    accessor<float4, 2, access::mode::read, access::target::image>
```

Comment:
Please, apply clang-format to this file.

Reply:
Moved check strings to begin of lines, but didn't split up it for correct filecheck work

Reply:
You can use `CHECK-SAME` if you need to split the line.
https://llvm.org/docs/CommandGuide/FileCheck.html#the-check-same-directive

Reply:
Thanks for help, I applied it, now patch fully clang-formated